src/Integration.Vsix/Suppression/LiveIssue.cs

```diff
@@ -39,7 +39,7 @@ internal class LiveIssue
             WholeLineText = wholeLineText;
 
             // SonarQube doesn't calculate hash for file-level issues
-            LineHash = startLine != 0 ? ChecksumCalculator.Calculate(WholeLineText) : "";
+            LineHash = wholeLineText != "" ? ChecksumCalculator.Calculate(WholeLineText) : "";
```

Comment:
Are you treating _wholeLineText == ""_ as an indication that the issue is a project level issue? Are there any rules that could be triggered on a genuinely empty line? If so, then the line hash for those genuinely empty lines will be wrong.

It might be better to pass a separate flag/have a separate static builder method to create a live issue for project-level issues.

Reply:
For now on, `wholeLineText` is empty on file-level and project-level issues. I can't find of a case where the line we report on would be empty.